src/components/Sidebar.js

```diff
@@ -1,33 +1,52 @@
-import { useEffect } from 'react';
-import styled from 'styled-components'
+import { useEffect, useState } from "react";
+import styled from "styled-components";
 
 const StyledSidebar = styled.div`
   border: 2px solid blue;
   width: 30%;
   display: flex;
   justify-content: center;
   align-items: center;
-`
+`;
+
+const StyledTrailListContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledTrailSelector = styled.div`
+  border: 2px solid lightgrey;
+  padding: 10px;
+  margin: 2px;
+  border-radius: 5px;
+`;
 
 const getTrails = async () => {
-  return fetch('data/trails/all-trails.json')
-  .then(res => res.json())
+  return fetch("data/trails/all-trails.json").then((res) => res.json());
 };
 
-
 export default function Sidebar() {
+  const [trailNames, setTrailNames] = useState([]);
+
   useEffect(() => {
-    getTrails()
-    .then(data => {
-      data.trails.forEach(trail => {
-        console.log(trail.name)
-      })
-    })
+    getTrails().then((data) => {
+      const namesArr = [];
+      data.trails.forEach((trail) => namesArr.push(trail.name));
+      setTrailNames(namesArr);
```

Comment:
I don't think creating an array specific to names is necessary. I think it is okay to just do:
```suggestion
     setTrails(trails)
```